praktikum/poglavlja/02_naredbe-selekcije/index.rst

```diff
@@ -0,0 +1,180 @@
+Naredbe selekcije
+==================
+
+Podeli svoje utiske na anketi `OVDE <https://docs.google.com/forms/d/1xG079mQsxOuhHc3l9vYXJmZzJjIlZGUFj7P-mkNwcv0>`_.
+
+Naredbe selekcije omogućuju da kontrolišemo tok izvršavanja programa.
+Ukoliko želimo da se određeni deo koda ne izvršava uvek, već samo pod određenim uslovima, koristimo naredbe selekcije.
+Deo koda koji je uslovljen naredbom selekcije će se izvršiti samo ukoliko je postavljeni uslov ispunjen.
+
+Napomena::
+
+  U programskom jeziku C se kao tačno podrazumeva sve što je različito od 0, dok je sve sto ima vrednost 0 netačno.
```

Comment:
Ne znam da li da se to doda pre ovoga ili kako se budu pojavljivali, ali potrebno je kratko o relacionim i logičkim operatorima